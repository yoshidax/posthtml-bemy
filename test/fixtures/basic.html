<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>

<body>
    <div class="_block --modifire">
        <div class="__element __element--modifire">
            <div class="_block-child">
                <div class="__element-1">1</div>
                <div class="__element-2">2</div>
                <div class="__element-3">3</div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
</body>

</html>
