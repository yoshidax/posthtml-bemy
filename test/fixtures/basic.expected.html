<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>

<body>
    <div class="block block--modifire">
        <div class="block__element block__element--modifire">
            <div class="block-child">
                <div class="block-child__element-1">1</div>
                <div class="block-child__element-2">2</div>
                <div class="block-child__element-3">3</div>
                <div class="something">something</div>
            </div>
        </div>
    </div>
</body>

</html>
